index.html

```diff
@@ -84,7 +93,7 @@ <h3>Multi-Post Stories</h3>
             <button type="button" class="project-button">See Project</button>
```

Comment:
As already highlighted, the button failed to meet an acceptable level for contrast ratio. You could use this [tool](https://www.aditus.io/button-contrast-checker/) to ensure that your buttons pass the checks. Kindly implement this as well.

Reply:
I have used the tool and adjusted contrast properties. Using the tool, I have been able to correct the errors shown by it. The main page color is changed to light grey and the button color property is changed to the appropriate one.